src/RageFileManager.h

```diff
@@ -27,6 +27,7 @@ class RageFileManager
 		vector<RString> const& ExtensionList, vector<RString> &AddTo,
 		bool bOnlyDirs= false, bool bReturnPathToo= false);
 	bool Move( const RString &sOldPath, const RString &sNewPath );
+	bool Copy( const std::string &fromPath, const std::string &toPath );
```

Comment:
Should we just use RString for consistency for now?

Reply:
I wouldn't. I would like to phase out RString and not using it for new code will make that easier. Otherwise we will wonder later whether a specific occurrence of RString can be replaced with std::string.

Reply:
+1 to deprecating RString!!

And auto_ptr in favor of unique_ptr. The now defunct "master" branch of StepMania had all these changes implemented which is so sad they never saw the light of day.